requirements.txt

```diff
@@ -1,4 +1,4 @@
 jc
 pytest
 python-ldap
-pytest-mh >= 1.0.3
+pytest-mh
```

Comment:
Shouldn't this be `pytest-mh >= 1.0.4`?

Reply:
Or maybe we need to release 1.0.5 once https://github.com/next-actions/pytest-mh/pull/11 is merged. I mean, there is kind of a tight relationship between this PR and that one.

Reply:
Okay we can wait for Pavel, but I don't think the `requirements.txt` commit needs to part of this PR so I removed it.

Reply:
Strictly speeking, we need to release pytest-mh 1.0.5 and bump the version here, please use `requirement: bump pytest-mh minimum version` as commit subject.

But our CI should always pull newest pytest-mh from its master branch, so no harm should be done in upstream CI. Downstream CI might get affected though.

Reply:
Okay, I added the commit to bump pytest-mh to >= 1.0.5.